app/Helpers/CSVHelper.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Helpers;
+
+use Illuminate\Foundation\Http\FormRequest;
+use League\Csv\Reader;
+
+class CSVHelper
+{
+    public static function validateFileAndReadToArray(FormRequest $request, array $headers): array
```

Comment:
do tej metody powinien być przekazywany plik, nie request. Nazwa metody na to wskazuje, plus w ten sposób będzie to bardziej uniwersalne. Lepiej będzie jak dla helpera nie będzie miało znaczenia, czy to jest request, czy z eventu, po prostu plik